app/jobs/question_mailer_job.rb

```diff
@@ -0,0 +1,7 @@
+class QuestionMailerJob < Que::Job
+  def run(question_id)
+    @question = Question.find(question_id)
+    @user     = User.find(@question.intended_respondent_id)
+    QuestionMailer.specified_user(@question).deliver_later
```

Comment:
We don't need to create additional jobs for mailers. ActionMailer already supports sending mails in background with `deliver_later` method http://api.rubyonrails.org/classes/ActionMailer/MessageDelivery.html#method-i-deliver_later